### ISSUE TITLE
Doubts in Train-validation split and SMD Parsing

### ISSUE THREAD

**Issue by pratham16cse** [imperial-qore/TranAD#27]
Hi @shreshthtuli,

Great work and thanks for making the benchmarks publically available. I have a couple of doubts --

1. On [line number 32 of main.py](https://github.com/imperial-qore/TranAD/blob/main/main.py#L32), it appears to me that only `machine-1-1_` is selected for parsing whereas the rest of the machines are not being considered. Whereas in Section 4.1 of the paper, you mention using machines `1-1,2-1,3-2 and 2-7`. 
2. In the last paragraph before section 4.1, it is mentioned that the training time series is divided into 80% training data and 20% validation data, and the training process stops when validation accuracy starts to decrease. However, 1) I did not find any training validation split anywhere in the code and 2) it is not clear how you evaluate validation accuracy when labels for training instances are not available (one such case is of SMD dataset).

Would be great if you could clarify the above points. Apologies if I have missed something in the code.

Thanks a lot,
Prathamesh

**Comment by shreshthtuli** [comment]
Thanks for your interestin our work. This is a refactored version of the code that uses only a single file for each dataset to generate results. We cumulate results over all files of a dataset (mentioned in the paper) to generate the required scores. To do this, you can run grid-search over hyperparameters for each file as described it the paper and generate corresponding results. We plan to make these hyperparameter values public as mentioned in #25.

**Comment by shreshthtuli** [comment]
We have added params in src/params.json.
